docs/md/se/jobs/11619.md

```diff
@@ -0,0 +1,35 @@
+# Agiler Software Engineer Java (m/w) 80-100%
+
+## Was dich erwartet
+
+Unser [Aufgabenspektrum](https://www.baloise.com/jobs/de/blog/baloise-jobcast/jobcast-it-engineer.html) ist klar. Der Fachbereich kommt mit einer Anforderung, wir besprechen diese gemeinsam und setzen dann entsprechend um.
+
+Klingt mega unspannend, ist es aber nicht, weil du bei uns die kreative Freiheit bekommst, eigene Lösungen zu finden. Wir sind so gar nicht top down, sondern vor allem auf Augenhöhe unterwegs. Du bist die IT-Kompetenz - vom Front- bis ins Backend, Java- bzw. Angular-basiert – und du bringst deine Vorstellung von einer möglichen Umsetzung ein.
```

Comment:
Java and Angular, one is a programming language, the other a SPA framework... 
-> Java and TypeScript, JEE and Angular

Reply:
done